portia/storage.py

```diff
@@ -1019,10 +1019,15 @@ def save_tool_call(self, tool_call: ToolCallRecord) -> None:
                     "latency_seconds": tool_call.latency_seconds,
                 },
             )
-        except Exception as e:
-            raise StorageError(e) from e
+        except Exception as e: # noqa: BLE001
+            logger().error(f"Error saving tool call to Portia Cloud (Not raising): {e}")
```

Comment:
```suggestion
            logger().error(f"Error saving tool call to Portia Cloud: {e}")
```
I don't think we need to say 'Not raising' here